Apps/WebClient/src/ClientApp/app/app.vue

```diff
@@ -1,196 +1,196 @@
-import { Getter } from "vuex-class";
-import Process, { EnvironmentType } from "@/constants/process.ts";
-
-// Load Bootstrap general plugins
-import {
-  LayoutPlugin,
-  NavPlugin,
-  NavbarPlugin,
-  ModalPlugin,
-  ButtonPlugin,
-  CardPlugin,
-  LinkPlugin,
-  FormPlugin,
-  FormInputPlugin,
-  FormCheckboxPlugin,
-  FormTextareaPlugin,
-  AlertPlugin,
-  SpinnerPlugin,
-  InputGroupPlugin,
-  PaginationNavPlugin,
-  TooltipPlugin
-} from "bootstrap-vue";
-Vue.use(LayoutPlugin);
-Vue.use(NavPlugin);
-Vue.use(NavbarPlugin);
-Vue.use(ModalPlugin);
-Vue.use(ButtonPlugin);
-Vue.use(CardPlugin);
-Vue.use(LinkPlugin);
-Vue.use(FormPlugin);
-Vue.use(FormInputPlugin);
-Vue.use(FormCheckboxPlugin);
-Vue.use(FormTextareaPlugin);
-Vue.use(AlertPlugin);
-Vue.use(SpinnerPlugin);
-Vue.use(InputGroupPlugin);
-Vue.use(PaginationNavPlugin);
-Vue.use(TooltipPlugin);
-
-// Load general icons
-import { config, library } from "@fortawesome/fontawesome-svg-core";
-// Prevent auto adding CSS to the header since that breaks Content security policies.
-config.autoAddCss = false;
-// Add font awesome styles manually
-import "@fortawesome/fontawesome-svg-core/styles.css";
-
-import {
-  faUser,
-  faUserCircle,
-  faAddressCard,
-  faUserSecret,
-  faEdit,
-  faChevronUp,
-  faChevronDown,
-  faSpinner,
-  faCheckCircle,
-  faTimesCircle,
-  faEllipsisV,
-  faPrint,
-  faAngleDoubleLeft,
-  faFileAlt,
-  faChartBar,
-  faCommentAlt,
-  faLock
-} from "@fortawesome/free-solid-svg-icons";
-library.add(
-  faUser,
-  faUserCircle,
-  faAddressCard,
-  faUserSecret,
-  faEdit,
-  faChevronUp,
-  faChevronDown,
-  faSpinner,
-  faCheckCircle,
-  faTimesCircle,
-  faEllipsisV,
-  faPrint,
-  faAngleDoubleLeft,
-  faFileAlt,
-  faChartBar,
-  faCommentAlt,
-  faLock
-);
-
-import HeaderComponent from "@/components/navmenu/navHeader.vue";
-import IdleComponent from "@/components/modal/idle.vue";
-import FooterComponent from "@/components/navmenu/navFooter.vue";
-import SidebarComponent from "@/components/navmenu/sidebar.vue";
-
-@Component({
-  components: {
-    NavHeader: HeaderComponent,
-    NavFooter: FooterComponent,
-    NavSidebar: SidebarComponent,
-    IdleComponent
-  }
-})
-export default class AppComponent extends Vue {
-  @Ref("idleModal") readonly idleModal?: IdleComponent;
-  @Getter("oidcIsAuthenticated", { namespace: "auth" })
-  oidcIsAuthenticated?: boolean;
-
-  private readonly host: string = window.location.hostname.toLocaleUpperCase();
-  private readonly isProduction: boolean =
-    Process.NODE_ENV == EnvironmentType.production &&
-    (this.host.startsWith("HEALTHGATEWAY") ||
-      this.host.startsWith("WWW.HEALTHGATEWAY"));
-
-  constructor() {
-    super();
-    console.log("Node ENV", Process.NODE_ENV);
-    console.log("host", this.host);
-  }
-
-  @Watch("isAppIdle")
-  public onIsAppIdleChanged(idle: boolean) {
-    if (idle && this.oidcIsAuthenticated) {
-      this.idleModal.show();
-    }
-  }
-}
-</script>
+<style lang="scss">
```

Comment:
this should not have changed. Can you check the line endings on your side?

Reply:
that's bizarre - just pulled in the latest from dev to fix it.